src/hardware/cpe-ubiquiti.md

```diff
@@ -27,17 +28,17 @@ Most devices come with a second `Management Radio` that is active for a short pe
 
 2. Once the WiFi connection is establish, open the browser and visit `https://192.168.172.1/`
 
-### Connect using an Ethernet Cable
+### Connect Using an Ethernet Cable
 
-Device can be connected to using an Ethernet cable. This method requires the knowledge of the device's IP address. If the device has been previously configured, and the IP address is known, adjust the steps below accordingly. If the IP address is not known, a factory reset can be performed by pressing the reset button for 30 seconds. This will erase all settings on the device and set the default IP address of 192.168.1.20 again.
+Device can be connected to using an ethernet cable. This method requires the knowledge of the device's IP address. If the device has been previously configured, and the IP address is known, adjust the steps below accordingly. If the IP address is not known, a factory reset can be performed by pressing the reset button for 30 seconds. This will erase all settings on the device and set the default IP address of 192.168.1.20 again.
```

Comment:
I think "Ethernet" actually should be capitalized?  Other than that this is good to go.

Reply:
```suggestion
Device can be connected to using an Ethernet cable. This method requires the knowledge of the device's IP address. If the device has been previously configured, and the IP address is known, adjust the steps below accordingly. If the IP address is not known, a factory reset can be performed by pressing the reset button for 30 seconds. This will erase all settings on the device and set the default IP address of 192.168.1.20 again.
```